### PR TITLE
Added Taverley shortcut to agility plugin

### LINKED ISSUE #6378: Add the Taverley outhouse stile to the agility shortcut plugin
image

### PR DESCRIPTION
**Pull request by MaxBartlett**
Closes #6378 

Image below:

screen shot 2018-11-07 at 5 05 29 pm

### CODE REVIEW

**Comment by MaxBartlett** [comment]
Don't know why that was so difficult. It should look good now.

File: runelite-client/src/main/java/net/runelite/client/plugins/agility/Obstacles.java [hunk 1]
```diff
@@ -217,7 +217,9 @@
 		// Weiss
 		LITTLE_BOULDER, ROCKSLIDE_33184, ROCKSLIDE_33185, NULL_33327, NULL_33328,
 		// Al-Kharid
-		BROKEN_WALL_33344, BIG_WINDOW
+		BROKEN_WALL_33344, BIG_WINDOW,
+		//Taverley
+		STILE
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/agility/Obstacles.java]: Is this the correct stile?

Comment (MaxBartlett) [reply runelite-client/src/main/java/net/runelite/client/plugins/agility/Obstacles.java]: Yes, it highlights the stile as shown in the image above.

Comment (deathbeam) [reply runelite-client/src/main/java/net/runelite/client/plugins/agility/Obstacles.java]: Needs rebase then

Comment (Nightfirecat) [reply runelite-client/src/main/java/net/runelite/client/plugins/agility/Obstacles.java]: This PR now changes 24 files and has 500 lines of diff...

Comment (MaxBartlett) [reply runelite-client/src/main/java/net/runelite/client/plugins/agility/Obstacles.java]: This was because I rebased.  Do you know how I can correct this?

Comment (Nightfirecat) [reply runelite-client/src/main/java/net/runelite/client/plugins/agility/Obstacles.java]: You've not rebased correctly then. You should try doing a hard reset to your last commit before the rebase, then doing `git fetch upstream && git rebase upstream/master`. Then you will have to resolve merge conflicts as your commits are applied upon that.

**Pull request merged**